vscode/src/registry.json

```diff
@@ -0,0 +1,64 @@
+{
+  "knownPackages": [
+    {
+      "name": "Signed",
+      "description": "Defines types and functions to work with signed qubit-based integers.",
+      "dependency": {
+        "github": {
+          "owner": "Microsoft",
```

Comment:
Being these are used to build GitHub URLs, "Microsoft" should probably be all lowercase.